src/test/java/com/notably/logic/suggestion/commands/NewSuggestionCommandTest.java

```diff
@@ -0,0 +1,45 @@
+package com.notably.logic.suggestion.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.model.Model;
+import com.notably.model.ModelManager;
+import com.notably.model.block.BlockModel;
+import com.notably.model.block.BlockModelImpl;
+import com.notably.model.suggestion.SuggestionModel;
+import com.notably.model.suggestion.SuggestionModelImpl;
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+
+public class NewSuggestionCommandTest {
+    private static AbsolutePath toRoot;
+    private static Model model;
+    private static NewSuggestionCommand newSuggestionCommand;
+
+    @Test
+    public void execute_responseText_test() {
+        // Set up model
+        BlockModel blockModel = new BlockModelImpl();
+        SuggestionModel suggestionModel = new SuggestionModelImpl();
+        ViewStateModel viewStateModel = new ViewStateModelImpl();
+        model = new ModelManager(blockModel, suggestionModel, viewStateModel);
+
+        // Set up path
+        AbsolutePath toRoot = AbsolutePath.fromString("/");
```

Comment:
Hmm, although this is executed only once, let's move this out to a `@BeforeAll` block. In the future, you won't need to do that again when you want to add more test cases